hack/testing/utils

```diff
@@ -221,20 +221,27 @@ function deploy_clusterlogging_operator() {
     local manifest=${repo_dir}/manifests
     local version=$(basename $(find $manifest -type d | sort -r | head -n 1))
     local csv=$manifest/$version/cluster-logging.*.clusterserviceversion.yaml
-    local es_img k_img c_img f_img op_img
+    local es_img k_img c_img f_img op_img    
+    local es_version="5"
+    
+    LOGGING_IMAGE_STREAM=${LOGGING_IMAGE_STREAM:-$version}
+    if [ "$LOGGING_IMAGE_STREAM" != "$version" ]; then
+        es_version="6"
```

Comment:
This is not needed.  This branch will always utilize ES6.  All components should be pegged to by kibana/elasticsearch6 with the exception of curator.  There is no "6" version of it

Reply:
Good catch!